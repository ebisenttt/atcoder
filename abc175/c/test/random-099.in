-704251786773329 36267419974843 542046513641986
