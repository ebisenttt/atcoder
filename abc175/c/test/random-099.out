162205273131343
